packages/style/scss/components/diff-viewer.scss

```diff
@@ -0,0 +1,29 @@
+.diff-viewer {
+    @import '~react-diff-view/style';
+
+    // configure the style of the library
+    --diff-background-color: var(--grey-10);
+    --diff-text-color: var(--grey-80);
+    --diff-font-family: $code-font-family;
```

Comment:
This doesn't seem to be applied

[`$code-font-family: source_code_pro_regular`](https://github.com/coveo/plasma/blob/master/packages/style/scss/variables.scss#L40)

Currently
![image](https://user-images.githubusercontent.com/35579930/169124523-86f06b93-0c43-40fd-abb7-4130eb07a54b.png)

With `'source_code_pro_regular'` applied

![image](https://user-images.githubusercontent.com/35579930/169123618-88d6a64f-12f7-4f8b-b92d-7dcbb6d64a1b.png)